riot/lib/ref.ml

```diff
@@ -1 +1 @@
-include Runtime.Core.Ref
```

Comment:
We should also move this file around and make it `symbol.ml`, so we export this on `riot.mli`.